run.sh

```diff
@@ -23,6 +23,36 @@ VBOX_MANAGE=/usr/bin/VBoxManage
 OCTETS="192.168."
 ORIGINAL_OCTETS=${OCTETS}"56"
 
+GB=1024
+SOFT_LIMIT=$(( 80 * GB ))
+HARD_LIMIT=$(( 30 * GB ))
+
+RED='\033[0;31m'
+YELLOW='\033[1;33m'
+NC='\033[0m'
+
+function check_available_disk_space() {
+    available_mb_str=$(df . --output=avail -B MB | grep -v "\W")
```

Comment:
@kouzant  Can I ask on this line - what platforms are you targeting for this to work on?  As the `df . --output=avail -B MB` part does not work on my OSX command line.  Thank you!